swapserverrpc/server.proto

```diff
@@ -677,4 +686,37 @@ message ServerAddressParameters {
 
     // The CSV expiry for the MuSig2 static address output.
     uint32 expiry = 2;
+}
+message ServerWithdrawRequest {
+    // The deposit outpoints the client wishes to close.
+    repeated ServerOutPoint outpoints = 1;
+
+    // The nonces that the client used to generate the coop close tx sigs.
+    repeated bytes client_nonces = 2;
+
+    // The address that the client wants to sweep the static address deposits
+    // to.
+    string client_sweep_addr = 3;
+
+    // The fee rate in sat/kw that the client wants to use for the sweep.
+    uint64 musig_tx_fee_rate = 4;
+}
+
+message ServerWithdrawResponse {
+    // The sweep sigs that the server generated for the htlc.
+    repeated bytes musig2_sweep_sigs = 1;
+
+    // The nonces that the server used to generate the sweepless sweep sigs.
+    repeated bytes server_nonces = 2;
+}
+
+message ServerOutPoint {
+    // Raw bytes representing the transaction id.
+    bytes txid_bytes = 1;
+
+    // Hex-encoded string representing the transaction id.
+    string txid_str = 2;
```

Comment:
These two fields seem to store the same information (txid) in different formats. Can we keep only one of them or use proto3's [oneof operator](https://protobuf.dev/programming-guides/proto3/#oneof)?

Reply:
Thanks for the suggestion, I used the `oneof` operator.